### ISSUE TITLE
Discard the gorm database

### ISSUE THREAD

**Issue by themoonstone** [qor/qor#212]
If I do not want to use gorm, just want to use qor, by calling api way to the data CRUD operation, how should we do? It is currently found that if you do not call "Admin: = admin.New (& qor.Config {})", do not add DB configuration will lead to panic

**Comment by aniruddha-mmt** [comment]
If you want to use Qor Admin, You have to pass DB instance(you can have a global instance for whole application) on which Qor provides you with the plug and play admin. 
Something like this .... 
// Initalization
Admin := admin.New(&admin.AdminConfig{DB: db.DB})

**Comment by thanateros** [comment]
I don't think it is the case that QOR Admin *needs* GORM to operate; it **does**, however, need a handle on the DB instance as mentioned above.

**Comment by cryptix** [comment]
You might also want to check out the [NoSQL issue](https://github.com/qor/admin/issues/113).

**Comment by sachin-puranik** [comment]
As I have understood till now, Just by replacing these 4 functions, we would be able to change the data storage and retrieval. But Ideally shouldn't it be done by porting QOR itself with such data base. 
Is there any possibility to use the GORM with say google DataStore which is a flat database.
How would db.AutoMigrate() work...
